roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/scan/RoqFrontMatterScanProcessor.java

```diff
@@ -412,8 +417,12 @@ private static Consumer<Path> addBuildItem(
             ZonedDateTime date = parsePublishDate(file, fm, config.dateFormat(), config.timeZone());
             final boolean noFuture = !config.future() && (collection == null || !collection.future());
             if (date != null && noFuture && date.isAfter(ZonedDateTime.now())) {
-                return;
-            }
+    LOGGER.warning("Ignoring document: " + file.getName() + " because it's scheduled for the future (" 
+                    + date.format(DateTimeFormatter.ISO_ZONED_DATE_TIME) + "). "
+                    + "To display future articles, use -Dsite.future=true");
```

Comment:
```suggestion
                    + "To display future articles, use -Dsite.future=true" + (collection == null ? "" :  " or -Dsite.collections." + collection.id() + ".future=true") );
```

It might not compile, this is added through GH but you get the idea..